java-squid/src/main/java/org/sonar/java/model/InternalSyntaxSpacing.java

```diff
@@ -17,28 +17,51 @@
  * License along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
  */
-package org.sonar.java.ast.visitors;
+package org.sonar.java.model;
 
-import com.google.common.base.Preconditions;
 import com.sonar.sslr.api.AstNode;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.sonar.sslr.api.AstNode'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)